swap/protocol_test.go

```diff
@@ -217,61 +215,60 @@ func TestHandshakeInvalidContract(t *testing.T) {
 	}
 }
 
-// TestEmitCheque is a full round of a cheque exchange between peers via the protocol.
-// We create two swap, for the creditor (beneficiary) and debitor (issuer) each,
-// and deploy them to the simulated backend.
-// We then create Swap protocol peers with a MsgPipe to be able to directly write messages to each other.
-// We have the debitor send a cheque via an `EmitChequeMsg`, then the creditor "reads" (pipe) the message
-// and handles the cheque.
+// TestEmitCheque tests the correct processing of EmitChequeMsg messages
+// One protocol tester is created which will receive the EmitChequeMsg
+// A second swap instance is created for easy creation of a chequebook contract which is deployed to the simulated backend
+// We send a EmitChequeMsg to the creditor which handles the cheque and sends a ConfirmChequeMsg
 func TestEmitCheque(t *testing.T) {
 	testBackend := newTestBackend()
-	defer testBackend.Close()
+	protocolTester, clean, err := newSwapTester(t, testBackend)
+	defer clean()
+	if err != nil {
+		t.Fatal(err)
+	}
+	swap := protocolTester.swap
```

Comment:
Can we please keep the `creditorSwap` terminology?

It is easier then to reason which swap it is when it stands on its own

Reply:
Makes sense. I changed the name back to `creditorSwap `.

Reply:
Thanks!